nixos/modules/installer/tools/nixos-install.sh

```diff
@@ -101,6 +101,7 @@ fi
 # Set the system profile to point to the configuration. TODO: combine
 # this with the previous step once we have a nix-env replacement with
 # a progress bar.
+echo "installing system..."
```

Comment:
Mhm this might be not required since the download already happen in `nix build` above?